### PR TITLE
[DOC] Update Romanian translation notes.txt

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
I propose some rules closer to the MSDN ones. Of course, I will modify everywhere.

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

**Comment by GeoB99** [comment]
That's some pretty radical change of the translation rules here... I would like to hear an opinion from @reactos-contributor in regard of this.

**Comment by AndreiMiloiu** [comment]
I also wait for his opinion. The idea of changes is to follow the MSDN rules as strictly as possible.

**Comment by reactos-contributor** [comment]
> That's some pretty radical change of the translation rules here... I would like to hear an opinion from @reactos-contributor in regard of this.

Yes, the change in rules is radical indeed and **I do not** support that. There are many more changes that I do not support, some of which also look pretty radical to me (like dropping the translation of „folder” to „dosar” in Romanian but keeping translation of „file” to „fișier”, i.e. the way Microsoft does it, but otherwise nonsensical to me). But then, the way I understand it, at this moment @AndreiMiloiu is the only one willing to dedicate time for up-keeping the Romanian resources in ReactOS and there isn't much point to oppose to anything.

P.S.: I did not had the chance to comment on many changes when they were in open PRs, as I only noticed them after they were committed. "Cc @reactos/lang-romanian" had effect only for a short while. Nowadays I do not get notification from that and the effort of adding such comment to PRs is pointless. I presume there was some settings change regarding notifications for that group, @GeoB99? And since the only currently active translator is not in it, what is the point of that group anyway?

**Comment by binarymaster** [comment]
> Nowadays I do not get notification from that and the effort of adding such comment to PRs is pointless. I presume there was some settings change regarding notifications for that group?

I think the problem is that this group can be pinged only by maintainers, i.e. a random contributor can't ping the teams.

> Yes, the change in rules is radical indeed and I do not support that. There are many more changes that I do not support, some of which also look pretty radical to me (like dropping the translation of „folder” to „dosar” in Romanian but keeping translation of „file” to „fișier”, i.e. the way Microsoft does it, but otherwise nonsensical to me).

It seems Andrei started to create PRs with the description "`Attendum to #6517`", which weren't meant for merging, at least before the documentation decisions are made here. 🤔

**Comment by AndreiMiloiu** [comment]
Every time I was open to discussions. As so many PRs from me were merged I understood that someone agreed with them so I continued my work. What I really want to do is to do an understandable translation that is as close as the MSDN's one. In this way, every user will be able to use ReactOS exactly as every NT version without any effort. I think that everything must be as familiar as possible in order to increase ReactOS's popularity. If we use an understandable and a familiar translation everyone has benefits both users and contributors. I DID NOT impose any rule. There were only proposals. However, I wouldn't agree that every idea of mine to be automatically rejected just because it's new here.

**Comment by JoachimHenze** [comment]
My view on that: Better Andrei grooms ro-RO than nobody taking really care of it.
I do like some of his changes, especially:
- the Confirma->OK
- Annulenza->Revocare
- grooming of accelerators to match MS
- „folder” to „dosar” is good if it leads to shorter texts
- "ș" to "ş" is good for added compatibility
- "ț" to "ţ" is good for added compatibility
- everything that shortens strings is always good

questionable is imho
- „file” to „fișier” as it is longer and has a "special character", and longer strings do typically cause us more headaches with fitting stuff into controls, but if that is what MS does, then it hard to argue against it

I do appreciate each contribution, and nobody aside him seems to really care for those translations.

**Comment by AndreiMiloiu** [comment]
Thank you very much @JoachimHenze and @binarymaster for your entire help and patience. Also, thank you for anyone else who helped me and was willing to do it. Thanks to you I know today how to translate Romanian resources in the right mode. Also, thanks to you I've got much experience since 2022 when I started translating ReactOS. If something is unclear now I know where to look in order to send you any screenshot you need. Hence, if everything is unclear related to my translations and accelerators, please ask me for a screenshot. I will send it to you in my free time. Also, I think it is a good idea to translate the rules in English in order to discuss them better. I will do it when I am going to have a little bit free time.

**Comment by GeoB99** [comment]
>  P.S.: I did not had the chance to comment on many changes when they were in open PRs, as I only noticed them after they were committed. "Cc @reactos/lang-romanian" had effect only for a short while. Nowadays I do not get notification from that and the effort of adding such comment to PRs is pointless. I presume there was some settings change regarding notifications for that group, @GeoB99? And since the only currently active translator is not in it, what is the point of that group anyway?

I don't know, probably GitHub toyed around with the groups feature or whatever, For what is worth, you're actually part of the group but if you don't get notifications then maybe send a ticket support at GitHub? /shrug

Anyway going back to the main topic, whilst the rules themselves are radical as you pointed out, getting closer to the MSDN guidelines is the only way to avoid any further debate over the translation rules again and again. Certainly we should not follow the "obvious" mistakes from Romanian translations on Windows but it's not like updating the rules every time there is an inconvenience does any favors. I myself am busy with important stuff such as the kernel and such. Not to dismiss @AndreiMiloiu's contributions or anything but I'm following my priorities. Reviewing his early PRs in the past gave me enough confidence that I can trust him on what he does so I tend to merge the translations when they're ready. 

With that said, I have no objections regarding this move. Not an ideal solution for every Romanian translator but IMHO it's time to finally release the anchor.

**Comment by AndreiMiloiu** [comment]
> My view on that: Better Andrei grooms ro-RO than nobody taking really care of it. I do like some of his changes, especially:
> 
> * the Confirma->OK
> * Annulenza->Revocare
> * grooming of accelerators to match MS
> * "ș" to "ş" is good for better compatibility
> * "ț" to "ţ" is good for better compatibility
> * everything that shortens strings is always good
> * „folder” to „dosar” is good if (and where) it leads to shorter texts
> 
> questionable is imho:
> 
> * „file” to „fișier” as it is longer and has a "special character", and longer strings do typically cause us more headaches with fitting stuff into controls, but if that is what MS does, then it hard to argue against it. It shouldn't be a priority though.
> * making strings longer for no obvious reasons (fixing an exisitng typo is a legit reason ofc, but touching existing stuff "just for the sake of consistency" is a weak justification and not worth touching it in most cases imho)
> 
> I do appreciate each contribution, and nobody aside him seems to really care for those translations.
> 
> The reason why I didn't approve this PR, is simply because I can't read romanian myself. It would actually be better if that document was written in english.

I translated the rules in English.

**Review by JoachimHenze — APPROVED** [review]

File: media/doc/Romanian translation notes.txt
Comment (JoachimHenze) [inline media/doc/Romanian translation notes.txt:8]:
```suggestion
 1. We use only the correct Romanian language diacritics.
     This means using lowercase ţ instead of ț.
     This means using lowercase ş instead of ș. The upper-case character to use for that is Ş.
```

File: media/doc/Romanian translation notes.txt
Comment (JoachimHenze) [inline media/doc/Romanian translation notes.txt:3]:
```suggestion
```
I think we can strip him, since there is none of the initial documents content left over.

**Review by GeoB99 — APPROVED** [review]
@AndreiMiloiu I'm fine with the newly introduced changes, I will merge it in the next few days if you do not have any further ado.

**Pull request merged**
